backend/tests/TestUtils/TestUtils.fs

```diff
@@ -1152,10 +1152,32 @@ let interestingDvals : List<string * RT.Dval * RT.TypeReference> =
      TDateTime)
     ("uuid", DUuid(System.Guid.Parse "7d9e5495-b068-4364-a2cc-3633ab4d13e6"), TUuid)
     ("uuid0", DUuid(System.Guid.Parse "00000000-0000-0000-0000-000000000000"), TUuid)
-    ("option", Dval.optionNone VT.int, TypeReference.option TInt)
-    ("option2", Dval.optionSome VT.int (Dval.int 15), TypeReference.option TInt)
+    ("option",
+     DEnum(
+       Dval.optionType,
+       Dval.optionType,
+       Dval.ignoreAndUseEmpty [ VT.int ],
+       "None",
+       []
+     ),
+     TypeReference.option TInt)
+    ("option2",
+     DEnum(
+       Dval.optionType,
+       Dval.optionType,
+       Dval.ignoreAndUseEmpty [ VT.int ],
+       "Some",
+       [ Dval.int 15 ]
+     ),
+     TypeReference.option TInt)
```

Comment:
Small note here - the Dval.option and Dval.result fns here became async (Ply) so don't work well here, and the point of the tests is to check _roundtripping_, not whether the Dval fns work correctly - so I opted to just create the DEnums manually here